src/js/breadcrumbs/specs/breadcrumbs-spec.js

```diff
@@ -0,0 +1,44 @@
+(function (define) {
+    'use strict';
+
+    define(['jquery',
+            'underscore',
+            '../breadcrumbs-view.js',
+            '../breadcrumbs-model.js'
+           ],
+           function($, _, BreadcrumbsView, BreadcrumbsModel) {
+
+               describe('Breadcrumbs component', function () {
```

Comment:
Nit: I tend to name specs for the class, so this would be ``describe('BreadcrumbsView', ...``